src/database/models/categories.rs

```diff
@@ -12,6 +12,7 @@ pub struct Loader {
     pub loader: String,
     pub icon: String,
     pub supported_project_types: Vec<String>,
+    pub supported_loaders: Vec<String>,
```

Comment:
This name is kinda off. I think "child_loaders" would work better. I think child loaders should also work with search - essentially, if a loader has child loaders, all of those loaders are added to the search document's loaders.